PlayCover/Utils/PlayTools.swift

```diff
@@ -83,7 +83,7 @@ class PlayTools {
             if result {
                 do {
                     try installPluginInIPA(exec.deletingLastPathComponent())
-                    try Shell.signApp(exec)
+                    Task { try await Shell.signApp(exec) }
```

Comment:
This app sign and the one in `removeFromApp` does not force the user to wait for the app to be fully signed, which causes some issues when opening the app without it being fully signed.

Reply:
Got it, this time I used GCD to deal with this kind of code that needs to be executed synchronously on some occasions.